Lib/idlelib/idle_test/test_searchengine.py

```diff
@@ -175,11 +175,13 @@ def test_getprog(self):
 
         engine.setpat('')
         Equal(engine.getprog(), None)
+        self.assertEqual(Mbox.showerror.message,
+                         'Error: Empty regular expression')
         engine.setpat('+')
         engine.revar.set(1)
         Equal(engine.getprog(), None)
         self.assertEqual(Mbox.showerror.message,
-                         'Error: nothing to repeat at position 0\nPattern: +')
```

Comment:
When I reviewed this file, written by GSOC student, I missed that this test passes because the buggy expected matches the buggy result of buggy code.